### PR TITLE
publish docs only for tags

### PR DESCRIPTION
**Pull request by vandot**

### CODE REVIEW

**Comment by vandot** [comment]
@Eknir do you agree to follow semver for tags?

**Review by Eknir — APPROVED** [review]
Thanks!

Approving, if this is the same semvar as we have in `bee`.

After this is merged, we will need to tag our current version, similarly to how we tagged Bee for the last release. @vandot , could you please do this, perhaps together with @significance ?

**Pull request merged**
